include/bl_storage.h

```diff
@@ -72,6 +128,27 @@ struct bl_storage_data {
 		uint32_t valid;
 		uint8_t hash[SB_PUBLIC_KEY_HASH_LEN];
 	} key_data[1];
+
+	/* Monotonic counters:
```

Comment:
So these are referring to the collections I guess. Lets use the same terms across the files so that it is easier to follow.